packages/core/src/Transfer/Transfer.stories.js

```diff
@@ -0,0 +1,454 @@
+/* eslint-disable react/prop-types */
+import React, { useState } from 'react'
+
+import {
+    SingleSelectField,
```

Comment:
This has to be imported from `@dhis2/ui-widgets`

Reply:
Good catch! :gloves:

Reply:
That's another one of those "we cannot import ui-widgets stuff in ui-core" problems, I'll solve it the same way I solved the InputField.